### PR TITLE
Release: 15.0.0

### LINKED ISSUE #964: Upcoming breaking changes (v15)
There are some dependencies that require a breaking changes, in terms of version bump. Therefore, we should plan for a v15 release of appkit that includes all those changes.

- [x] eslint / eslint-airbnb #911 
- [x] react >= 16.9 #966 
- [x] @testing-library >= 9 #956 
- [x] apollo hooks #801

### PR DESCRIPTION
**Pull request by tdeekens**
#### Summary

This is an aggregation of us working towards the v15 release of app-kit.

- Closes #964

### CODE REVIEW

**Comment by emmenko** [comment]
Btw, we don't need to keep this PR open yet, we can open it once we merge all the PRs for the release in the v15 branch

**Comment by tdeekens** [comment]
Thanks, I know. However I like to keep things visible.

**Comment by emmenko** [comment]
I think we should be good to go here. @tdeekens anything else missing from your side that you would like to include in this release?

**Comment by emmenko** [comment]
Yes, sounds good. I can also draft the changelog in here

**Comment by emmenko** [comment]
FYI: I'm going to publish an alpha version later

**Comment by emmenko** [comment]
I drafted the release notes for v15. Please have a look @tdeekens @montezume

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,77 @@
...
+  "resolutions": {
+    "react": "16.9.0",
+    "**/react": "16.9.0",
+    "react-test-renderer": "16.9.0",
+    "**/react-test-renderer": "16.9.0"
+  }
+}
+```
+
+Another important change is the support for `react-apollo` `v3`, which ships with React Hooks support.
```
Comment (montezume) [inline CHANGELOG.md]:
```suggestion
Another important change is support for `react-apollo` `v3`, which ships with React Hooks support.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,77 @@
+## [15.0.0](https://github.com/commercetools/merchant-center-application-kit/compare/v14.5.0...v15.0.0) (2019-08-23)
+
+This release introduces some **breaking changes** related to minimal dependency versions.
```
Comment (montezume) [inline CHANGELOG.md]: does this mean we need to use `react`, `react-dom` version 19?

Comment (emmenko) [reply CHANGELOG.md]: Yes, we should use React 16.9

**Comment by emmenko** [comment]
I'll release a new alpha version: `15.0.0-alpha.2`

**Comment by emmenko** [comment]
Alright, things are looking good in the MC apps, so we will proceed with the release of v15 today. If you have any last wishes, this is the time to raise your hand 😉

**Pull request merged**